src/functions-and-arrays.js

```diff
@@ -1,16 +1,65 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (a, b) {
+  if (a > b){
+    return a;
+  } 
+  else {
+    return b;
+  }
+}
+
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+let wordsLength = 0;
+let longestWord;
+
+
+function findLongestWord () {
+  for(let i=0;i<words.length;i++) {
+    if (words[i].length > wordsLength) {
+      wordsLength = words[i].lenght;
+      longestWord = words[i];
+    }
+  }
+  return longestWord;
+}
+
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+let sum = 0;
+let totalSum = 0;
+
+
+function sumNumbers () {
+  for (let i=0; i<=numbers.length-1; i++){
+    let sum = numbers[i] + totalSum;
```

Comment:
Nice job!
Few comments here:

1. here you could have used only sum to return the whole sum of the numbers. total sum was not needed.
2. See when you were calculating sum you were recreating a new sum in every loop `let sum` but if you just did this `sum= sum+numbers[i]` you could still have gotten the total at the end of the loop and save your self some code :)
and then in the return you just `return sum`



also you could have calculated sum like this sum